build-docker-plotman.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
```

Comment:
Do we really want to think about not-bash?

```suggestion
#!/bin/bash
```